### PR TITLE
libsoup: update to 2.68.2 and build with meson

### PR DESCRIPTION
**Pull request by flyn-org**
Signed-off-by: W. Michael Petullo <mike@flyn.org>

Maintainer: me
Compile tested: x86_64 master

Description:
Thank you for #10043!

### CODE REVIEW

**Comment by neheb** [comment]
Error log: https://5915-20307838-gh.circle-artifacts.com/0/home/build/build_dir/logs/package/feeds/packages/libsoup/compile.txt

Missing build dependency on glib2/host looks like.

File: libs/libsoup/Makefile [hunk 1]
```diff
@@ -6,24 +6,25 @@
...
-PKG_HASH:=3f3718623338f1bd7d7899dae2bdb613348212d59999a27432120afc1435ff04
+PKG_SOURCE_URL:=@GNOME/$(PKG_NAME)/2.68
+PKG_HASH:=51ad3001a946fe3bcf29b692dc9ffe05cdf702ea6ca0ee8c3099a99a2f4e3933
 
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=COPYING
 PKG_CPE_ID:=cpe:/a:gnome:libsoup
 
-PKG_BUILD_DEPENDS:=intltool/host
+PKG_BUILD_DEPENDS:=intltool/host meson/host
```
Comment (neheb) [inline libs/libsoup/Makefile]: I don't think intltool/host is needed now that meson is used.

File: libs/libsoup/Makefile [hunk 2]
```diff
@@ -6,24 +6,25 @@
...
+PKG_VERSION:=2.68.2
+PKG_RELEASE:=1
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_SOURCE_URL:=@GNOME/$(PKG_NAME)/2.65
-PKG_HASH:=3f3718623338f1bd7d7899dae2bdb613348212d59999a27432120afc1435ff04
+PKG_SOURCE_URL:=@GNOME/$(PKG_NAME)/2.68
+PKG_HASH:=51ad3001a946fe3bcf29b692dc9ffe05cdf702ea6ca0ee8c3099a99a2f4e3933
 
 PKG_LICENSE:=GPL-2.0
```
Comment (neheb) [inline libs/libsoup/Makefile]: This should be LGPL-2.1-or-later

**Comment by flyn-org** [comment]
@neheb, I addressed your comments. Thank you.

**Comment by dhewg** [comment]
A little late, but fyi:
* `PKG_FIXUP:=autoreconf` can obviously get nuked
* `PKG_BUILD_PARALLEL` is a gnu make thing, it's not required when using meson

**Pull request merged**